js/templates/shared/toggle-button.js

```diff
@@ -0,0 +1,18 @@
+const bel = require('bel');
+
+module.exports = function (isActiveBoolean, klass, dataKey) {
+
+    // make `klass` and `dataKey` optional:
+    klass = klass || '';
+    dataKey = dataKey || '';
+
+    return bel`
+    <button class="toggle-button toggle-button--is-active-${isActiveBoolean} ${klass}"
+            data-key="${dataKey}"
+            type="button">
+        <div class="toggle-button__bg">
+        </div>
+        <div class="toggle-button__knob"></div>
+    </button>`;
```

Comment:
i think it just has to do with the way we each use our keyboards/mice. my guess is you maybe tab thru things whereas i just use mouse? when you tab thru elements, chrome auto-detects this and creates a css :focus state by default. that chrome setting is now overridden by our resets in base.css.